deploy/README.md

```diff
@@ -68,9 +77,146 @@ The diagram below illustrates the components of the Kubernetes collection soluti
 * **kube-state-metrics.** Listens to the Kubernetes API server; generates metrics about the state of the deployments, nodes, and pods in the cluster; and exports the metrics as plaintext on an HTTP endpoint listen port.
 * **Prometheus deployment.** Scrapes the metrics exposed by the `node-exporter` add-on for Kubernetes and the `kube-state-metric`s component; writes metrics to a port on the Fluentd deployment.
 * **Fluentd deployment.** Forwards logs and metrics to HTTP sources on a hosted collector. Includes multiple Fluentd plugins that parse and format the metrics and enrich them with metadata.
-* **Events fluentd deployment.** Forwards events to an HTTP source on a hosted collector.
+* **Events Fluentd deployment.** Forwards events to an HTTP source on a hosted collector.
 
-## Before you start
+## Minimum Requirements
+
+Name | Version
+-------- | -----
+K8s with EKS | 1.13.8
+|| 1.11.10
+K8s with Kops | 1.13.10-k8s<br>1.13.0-kops
+|| 1.12.8-k8s<br>1.12.2-kops
+||1.10.13-k8s<br>1.10.0-kops
+K8s with GKE | 1.12.8-gke.10<br>1.12.7-gke.25<br>1.11.10-gke.5
+K8s with AKS | 1.12.8
+Helm | 2.4.13 (Linux)
+kubectl | 1.15.0
+
+## Support Matrix
+
+The following matrix displays the tested package versions for our Helm chart.
+
+Sumo Logic Helm Chart | Prometheus Operator | Fluent Bit | Falco
+|:-------- |:-------- |:-------- |:--------
+0.4.0 | 6.2.1 | 2.4.4 | 1.0.5
+
+## Installation with Helm
+
+Our Helm chart deploys Kubernetes resources for collecting Kubernetes logs, metrics, and events; enriching them with deployment, pod, and service level metadata; and sends them to Sumo Logic.
+
+### How to install when no Prometheus exists
```

Comment:
nit: Since these steps are the default installation steps, and the `with existing prometheus operator` and `with existing standalone prometheus` are kind of "extra" stuff, maybe we can just call this section `How to install`? not sure how others feel about this

Reply:
I was following Bryan's structure guide, so if wants to change it, okay. :)